tajo-core/src/main/java/org/apache/tajo/querymaster/Repartitioner.java

```diff
@@ -725,13 +725,8 @@ public static void scheduleRangeShuffledFetches(TaskSchedulerContext schedulerCo
       }
     }
 
-    boolean ascendingFirstKey = sortSpecs[0].isAscending();
     SortedMap<TupleRange, Collection<FetchImpl>> map;
-    if (ascendingFirstKey) {
-      map = new TreeMap<TupleRange, Collection<FetchImpl>>();
-    } else {
-      map = new TreeMap<TupleRange, Collection<FetchImpl>>(new TupleRange.DescendingTupleRangeComparator());
```

Comment:
I'm sorry for late response. Descending order is for the future feature. So, it is not necessary right now. Later, it can be added again when necessary.